docs/docs/concepts/05-building-blocks/05-hardware-interfaces.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 5
+---
+
+# Hardware Interfaces
```

Comment:
I believe 'Hardware interfaces' was not capitalized earlier. Shouldn't it be 'Hardware interfaces' then?